javascript/javascript1/week3/CactusI0Interactive.js

```diff
@@ -0,0 +1,66 @@
+const activities =[];
+const emptyArray = [];
+const options = { weekday: 'long', year: 'numeric', month: 'long', day: 'numeric' };
+const today  = new Date();
+const oneActivity = [{ date:today.toLocaleDateString("en-US"), activity: 'Youtube', duration: 30 }]
+function addActivity(activity, duration) {
+    if (isNaN(duration) || duration === "") {
+        console.log("Please put a valid duration")
+    }
+    else {
+        return activities.push({date:today.toLocaleDateString("en-US"), activity:activity, duration:duration})
+    }
+}
+addActivity("Youtube", 30);
+addActivity("Movie", 170);
+addActivity("Talking with parents", 45);
+
+function showStatus(array) {
+    let x = 0;
```

Comment:
Try to name your variables in a way that suggests what is stored in them. It's a bood way to make your code more readable. in this case `x` could be called `totalActivitiesDuration` or somthing like that. 
How else could you call `n` variable so it is clear from the start what we will put there - imagine reading this after a month when you do not remember what was in the assignment. Will you be able to tell what is `x` quickly? :)